FhirToDataLake/deploy/templates/ContainerApp/DeployDicomPipelineToContainerApp.json

```diff
@@ -127,7 +127,7 @@
     "jobInfoQueueName": "[concat(variables('pipelineName'), 'jobinfoqueue')]",
     "jobInfoTableName": "[concat(variables('pipelineName'), 'jobinfotable')]",
     "metadataTableName": "[concat(variables('pipelineName'), 'metadatatable')]",
-    "resourceTags": {"FhirAnalyticsPipeline": "DicomToDataLake", "FhirSchemaVersion": "v0.6.0"},
+    "resourceTags": {"FhirAnalyticsPipeline": "DicomToDataLake", "FhirSchemaVersion": "v0.6.1"},
```

Comment:
Not related to this PR and just noticed that the tags use "FHIR" in DICOM pipeline. Maybe we can try to refine the tags later.

Reply:
Sure, I think those should be updated as we current rename to Health Data Analytics Connector, let us discuss this later offline.